server/handler/base.go

```diff
@@ -204,14 +236,20 @@ func (b *Base) EvaluateFetchedConfig(ctx context.Context, prctx pull.Context, cl
 		statusState = "error"
 		statusDescription = "All rules were skipped. At least one rule must match."
 	default:
-		return errors.Errorf("evaluation resulted in unexpected state: %s", result.Status)
+		return errors.Errorf("Evaluation resulted in unexpected status: %s", result.Status)
 	}
 
-	if err := b.PostStatus(ctx, prctx, client, statusState, statusDescription); err != nil {
-		return err
+	return b.PostStatus(ctx, prctx, client, statusState, statusDescription)
+}
+
+func (b *Base) RequestReviewsForResult(ctx context.Context, prctx pull.Context, client *github.Client, result common.Result) error {
+	logger := zerolog.Ctx(ctx)
+
+	if prctx.IsDraft() {
+		return nil
 	}
 
-	if statusState == "pending" && !prctx.IsDraft() {
+	if result.Status == common.StatusPending {
```

Comment:
You could probably combine this with the new draft check and return early, unnesting the rest of this function:

```go
if prctx.IsDraft() || result.Status != common.StatusPending {
  return nil
}
```